### PR TITLE
Cut 0.15.0 rc.0

### PR DESCRIPTION
**Pull request by bwplotka**
+ some extra fixes in separate commits (merge without squash!)

### CODE REVIEW

**Comment by povilasv** [comment]
there is a bunch of code changes in this one?

**Comment by bwplotka** [comment]
Yes on purpose (:

**Comment by bwplotka** [comment]
Check each commits to learn more please

**Comment by bwplotka** [comment]
PTAL @povilasv @GiedriusS @metalmatze @squat

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,61 +7,69 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-
----
-sse_config:
-  type: SSE-S3
-```
+## [v0.15.0-rc.0](https://github.com/thanos-io/thanos/releases/tag/v0.15.0-rc.0) - 2020.08.26
 
+Highlights:
+* Added new Thanos component: [Query Frontend](https://thanos.io/tip/components/query-frontend/) responsible for response caching and
+query scheduling & parallelization (based on Cortex Query Frontend).
```
Comment (krasi-georgiev) [inline CHANGELOG.md:18]:
```suggestion
* Added new Thanos component: [Query Frontend](https://thanos.io/tip/components/query-frontend/) responsible for response caching,
query scheduling and parallelization (based on Cortex Query Frontend).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,61 +7,69 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+* Added various new, improved UIs to Thanos based on React: Querier' BuildInfo & Flags, Ruler UI, BlockViewer.
+* Optimized Sidecar, Store, Receive, Ruler data retrieval with new TSDB ChunkIterator, caping chunks to 120 samples, fixed various leaks.
```
Comment (krasi-georgiev) [inline CHANGELOG.md:20]:
```suggestion
* Optimized Sidecar, Store, Receive, Ruler data retrieval with new TSDB ChunkIterator, capping chunks to 120 samples, fixed various leaks.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,61 +7,69 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- [#2957](https://github.com/thanos-io/thanos/pull/2957) Rule: *breaking :warning:* Now sets all of the relevant fields properly; avoids a panic when `/api/v1/rules` is called and the time zone is _not_ UTC; `rules` field is an empty array now if no rules have been defined in a rule group.
+Thanos Rule's `/api/v1/rules` endpoint no longer returns the old, deprecated `partial_response_strategy`. The old, deprecated value has been fixed to `WARN` for quite some time. _Please_ use `partialResponseStrategy`.
 - [#2976](https://github.com/thanos-io/thanos/pull/2976) Query: Better rounding for incoming query timestamps.
 - [#2929](https://github.com/thanos-io/thanos/pull/2929) Mixin: Fix expression for 'unhealthy sidecar' alert and also increase the timeout for 10 minutes.
 - [#3024](https://github.com/thanos-io/thanos/pull/3024) Query: consider group name and file for deduplication
-- [#3064](https://github.com/thanos-io/thanos/pull/3064) s3: Add SSE/SSE-KMS/SSE-C configuration.
+- [#3012](https://github.com/thanos-io/thanos/pull/3012) Ruler,Receiver: TSDB not deletes blocks in atomic way.
```
Comment (krasi-georgiev) [inline CHANGELOG.md:42]:
```suggestion
- [#3012](https://github.com/thanos-io/thanos/pull/3012) Ruler,Receiver: Fix TSDB to delete blocks in atomic way.
```

**Review by krasi-georgiev — COMMENTED** [review]
LGTM with few small nits

**Pull request merged**